code/src/constants/deliveryMethods.ts

```diff
@@ -4,10 +4,12 @@ import ukrPostaImage from "@/assets/images/dashboard/ukr-posta.webp";
 export const deliveryMethods = {
   NOVA_POST: {
     translationKey: "dashboardTabs.orders.filters.novaPost",
-    image: novaPostaImage
+    image: novaPostaImage,
+    value: "NOVA"
   },
   UKR_POST: {
     translationKey: "dashboardTabs.orders.filters.ukrPost",
-    image: ukrPostaImage
+    image: ukrPostaImage,
+    value: "UKRPOSHTA"
```

Comment:
We need to use one approach. We use `ukr post` as key and `UKRPOSHTA` as value

Reply:
discussed in voice